packages/devtools/src/screens/ChatScreen/ChatScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC, useContext, useCallback, useState } from 'react';
+import { ChatContext } from '../../stores/ChatStore';
+import { useClasses } from './ChatScreen.styles';
```

Comment:
nit: can we order the imports like is done [her](https://github.com/aacebo/teams-sdk-js/blob/main/packages/dev/src/devtools/plugin.ts#L1), ordered from top to bottom with system imports being first (react, node) followed by 3rd party packages, then finally our internal imports (using relative paths as you already have it). 

```typescript
import React from 'react';
import * as uuid from 'uuid';

import XComponent from '../../components/X';
import { MyLocalType } from './MyLocalType';
```

this is to make it clearer throughout the codebase when ready through imports which are 3rd party and which are not mostly, but also in my case at least, makes the code easier to read in general.